.travis.yml

```diff
@@ -22,7 +22,6 @@ install:
   - echo eclipsePlugin.dir=$(pwd)/eclipse/plugins > eclipsePlugin/local.properties
 
 script:
-  - ./gradlew assemble
-  # Manually switch to JDK9 for tests if needed
+  # Manually switch to JDK9 if needed
   - if [[ $JDK_FOR_TEST == "oraclejdk9" ]]; then remove_dir_from_path $JAVA_HOME/bin; export JAVA_HOME=/usr/lib/jvm/java-9-oracle; export JAVA_HOME=/usr/lib/jvm/java-9-oracle; fi
-  - ./gradlew test smoketest
+  - ./gradlew assemble test smoketest -S
```

Comment:
If anybody doing the review .. `-S` is just printing the stacktrace in case of failure. https://docs.gradle.org/current/userguide/gradle_command_line.html